src/test/EigenPod.t.sol

```diff
@@ -237,7 +245,9 @@ contract EigenPodTests is ProofParsing, EigenPodPausingConstants {
              strategyManager
         );
 
-        cheats.deal(address(podOwner), 5*stakeAmount);     
+        cheats.deal(address(podOwner), 5*stakeAmount);   
+
+        emit log("WHT is happ");  
```

Comment:
looks like there's a few logs you forgot to delete? 😄

Reply:
lol removed